.github/workflows/trailing_commas.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
```

Comment:
This file does not comply with pylint, pydocstyle, pycodestyle and pep8 . Please run those commands and fix the console error.